quidel/delphi_quidel/run.py

```diff
@@ -21,7 +27,9 @@
 
 def run_module():
     """Run Quidel flu test module."""
+    start_time = time.time()
```

Comment:
this seems like something that could be a decorator, though not sure quite how to get around the logger being initialized within the function....